app/controllers/api/functions.php

```diff
@@ -1599,13 +1601,14 @@
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_EXECUTION)
     ->param('functionId', '', new UID(), 'Function ID.')
-    ->param('body', '', new Text(0, 0), 'HTTP body of execution. Default value is empty string.', true)
+    ->param('body', '', new Text(10485760, 0), 'HTTP body of execution. Default value is empty string.', true)
```

Comment:
Why 10MB, and not 20MB like on executor?

Reply:
The max payload size in Appwrite is 12 MB - so made it 10 MB here to keep some margin for metadata.
https://github.com/appwrite/appwrite/blob/e297371aa4c87a2347fb57e0510e65095a18c95f/app/http.php#L35C2-L35C13